### ISSUE TITLE
unit.test_loader.LoaderGlobalsTest.test_states test failures

### ISSUE THREAD

**Issue by garethgreenaway** [saltstack/salt#53014]
2019.2 failed [salt-centos-7-py2](https://jenkinsci.saltstack.com/job/2019.2/job/salt-centos-7-py2/79/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-centos-7-py2-m2crypto](https://jenkinsci.saltstack.com/job/2019.2/job/salt-centos-7-py2-m2crypto/80/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-centos-7-py2-transport](https://jenkinsci.saltstack.com/job/2019.2/job/salt-centos-7-py2-transport/73/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-centos-7-py3](https://jenkinsci.saltstack.com/job/2019.2/job/salt-centos-7-py3/75/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-centos-7-py3-m2crypto](https://jenkinsci.saltstack.com/job/2019.2/job/salt-centos-7-py3-m2crypto/82/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-centos-7-py3-pycryptodomex](https://jenkinsci.saltstack.com/job/2019.2/job/salt-centos-7-py3-pycryptodomex/81/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-centos-7-py3-transport](https://jenkinsci.saltstack.com/job/2019.2/job/salt-centos-7-py3-transport/73/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-fedora-28-py3](https://jenkinsci.saltstack.com/job/2019.2/job/salt-fedora-28-py3/79/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-fedora-29-py3](https://jenkinsci.saltstack.com/job/2019.2/job/salt-fedora-29-py3/79/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-opensuse-15-py2](https://jenkinsci.saltstack.com/job/2019.2/job/salt-opensuse-15-py2/67/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-opensuse-15-py3](https://jenkinsci.saltstack.com/job/2019.2/job/salt-opensuse-15-py3/64/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-ubuntu-1604-py2](https://jenkinsci.saltstack.com/job/2019.2/job/salt-ubuntu-1604-py2/78/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-ubuntu-1604-py2-m2crypto](https://jenkinsci.saltstack.com/job/2019.2/job/salt-ubuntu-1604-py2-m2crypto/78/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-ubuntu-1604-py2-pycryptodomex](https://jenkinsci.saltstack.com/job/2019.2/job/salt-ubuntu-1604-py2-pycryptodomex/79/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-ubuntu-1604-py2-transport](https://jenkinsci.saltstack.com/job/2019.2/job/salt-ubuntu-1604-py2-transport/74/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-ubuntu-1604-py3](https://jenkinsci.saltstack.com/job/2019.2/job/salt-ubuntu-1604-py3/77/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-ubuntu-1604-py3-m2crypto](https://jenkinsci.saltstack.com/job/2019.2/job/salt-ubuntu-1604-py3-m2crypto/78/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-ubuntu-1604-py3-pycryptodomex](https://jenkinsci.saltstack.com/job/2019.2/job/salt-ubuntu-1604-py3-pycryptodomex/82/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-ubuntu-1604-py3-transport](https://jenkinsci.saltstack.com/job/2019.2/job/salt-ubuntu-1604-py3-transport/73/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-ubuntu-1804-py2](https://jenkinsci.saltstack.com/job/2019.2/job/salt-ubuntu-1804-py2/79/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-ubuntu-1804-py3](https://jenkinsci.saltstack.com/job/2019.2/job/salt-ubuntu-1804-py3/81/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-windows-2016-py3](https://jenkinsci.saltstack.com/job/2019.2/job/salt-windows-2016-py3/24/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states), [salt-windows-2019-py2](https://jenkinsci.saltstack.com/job/2019.2/job/salt-windows-2019-py2/63/testReport/junit/unit.test_loader/LoaderGlobalsTest/test_states)
---

u'__pillar__' not found in ['unicode_literals', 'absolute_import', 'subprocess', '__path__', 'depends', 'ignores_kwargs', '_DeprecationDecorator', 'LOG_LEVELS', 'wraps', 'log', 'errno', 'six', 'state', '__package__', 'ensure_unicode_args', 'SaltConfigurationError', 'memoize', 'signature', '__doc__', 'defaultdict', '__builtins__', '__file__', 'inspect', 'sys', 'timing', '__name__', '_IsDeprecated', '_WithDeprecated', 'print_function', 'jinja', 'CommandExecutionError', 'with_deprecated', 'logging', 'is_deprecated', 'Depends', 'IS_WINDOWS', 'time', 'path', 'salt']

```

Traceback (most recent call last):
  File "c:\users\admini~1\appdata\local\temp\kitchen\testing\tests\unit\test_loader.py", line 1075, in test_states
    self._verify_globals(salt.loader.states(self.master_opts, {}, {}, {}))
  File "c:\users\admini~1\appdata\local\temp\kitchen\testing\tests\unit\test_loader.py", line 999, in _verify_globals
    self.assertIn(name, list(item.keys()))
AssertionError: u'__pillar__' not found in ['unicode_literals', 'absolute_import', 'subprocess', '__path__', 'depends', 'ignores_kwargs', '_DeprecationDecorator', 'LOG_LEVELS', 'wraps', 'log', 'errno', 'six', 'state', '__package__', 'ensure_unicode_args', 'SaltConfigurationError', 'memoize', 'signature', '__doc__', 'defaultdict', '__builtins__', '__file__', 'inspect', 'sys', 'timing', '__name__', '_IsDeprecated', '_WithDeprecated', 'print_function', 'jinja', 'CommandExecutionError', 'with_deprecated', 'logging', 'is_deprecated', 'Depends', 'IS_WINDOWS', 'time', 'path', 'salt']
		
```

**Comment by KChandrashekhar** [comment]
Test is failing on 2019.2 branch and not on 2019.2.1 so it will be fixed in 2019.2.2 release

**Comment by twangboy** [comment]
Failing on develop:
May 21, 2019 5:42:00 PM - https://jenkinsci.saltstack.com/job/develop/job/salt-windows-2019-py2/95/
May 21, 2019 9:43:00 PM - https://jenkinsci.saltstack.com/job/develop/job/salt-windows-2016-py2/49/
